test/parallel/test-tick-processor.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+var fs = require('fs');
+var assert = require('assert');
+var path = require('path');
+var cp = require('child_process');
+var common = require('../common');
+
+common.refreshTmpDir();
+
+var prevDir = process.cwd();
+process.chdir(common.tmpDir);
+var f = cp.execFileSync(process.execPath, ['-prof', '-pe',
+    'function fib(n) { return n < 2 ? n : fib(n-1) + fib(n-2); }; fib(40);']);
```

Comment:
I speculate this is going to time out on the low-powered buildbots.  Maybe move the test to the test/pummel directory?